AsBuiltReport.Fortinet.Fortigate.json

```diff
@@ -8,14 +8,16 @@
         "ShowHeaderFooter": true,
         "ShowTableCaptions": true
     },
-    "Options": {
-
-    },
+    "Options": {},
     "InfoLevel": {
-		"_comment_": "Please refer to the AsBuiltReport project contributing guide for information about how to define InfoLevels.",
-        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive"
+        "_comment_": "Please refer to the AsBuiltReport project contributing guide for information about how to define InfoLevels.",
+        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive",
```

Comment:
Update this based on the number of InfoLevels you intend to have. You don't have to have 5. Minimum would be 3. `0 = Disabled, 1 = Enabled / Summary, 2 = Detailed`